app/serializers.py

```diff
@@ -493,3 +493,58 @@ class LeaveUpdateSerializer(serializers.ModelSerializer):
     class Meta:
         model = models.Leave
         fields = ("status", "hr_comment")
+
+
+class VerifyEmailSerializer(serializers.Serializer):
+    email = serializers.EmailField(required=True)
+
+
+class OrganizationSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = models.Organization
+        fields = "__all__"
+
+
+class OwnerOnBoardingSerializer(serializers.ModelSerializer):
+    organization = OrganizationSerializer()
+
+    class Meta:
+        model = models.User
+        fields = ("first_name", "last_name", "email", "organization")
+
+    @transaction.atomic
+    def create(self, validated_data):
+        organization = validated_data.pop("organization")
+        organization_serialzier = OrganizationSerializer(data=organization)
+        organization_serialzier.is_valid(raise_exception=True)
+        organization_serialzier.save()
```

Comment:
```suggestion
        organization = org_serializer.save()
```
`save()` returns the instance created.